tests/integration/json-schema-files/redis-schema-metrics.json

```diff
@@ -186,8 +186,6 @@
                   "db.syncPartialOk",
                   "event_type",
                   "net.blockedClients",
-                  "net.clientBiggestInputBufBytes",
```

Comment:
The new version of the redis image did not included these metrics, therefore I marked them as non-required